recipes/rtmidi/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class RtMidiConan(ConanFile):
+    name = "rtmidi"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://www.music.mcgill.ca/~gary/rtmidi/"
+    description = "Realtime MIDI input/output"
+    topics = ("midi")
+    license = "MIT+send-patches-upstream"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "pkg_config"
+    exports_sources = "CMakeLists.txt", "patches/*"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    _cmake = None
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        if self.settings.os == "Linux":
+            self.requires("libalsa/1.2.4")
```

Comment:
Does the cmake accept definitions for libalsa?
Because it can pick up a system library.

Reply:
Never mind, dumb suggestion.
Alsa is mandatory on Linux.